### ISSUE TITLE
Docker won't start with existing bridge and fixed CIDR of 172.17.0.0/24

### ISSUE THREAD

**Issue by jkds** [docker/docker#10382]
Hi,

I recently was configuring three Ubuntu hosts to talk over the same subnet i.e.
172.17.x.x
and ran into problems.
I have a pre-created bridge which has the following configuration:

```
auto docker0
iface docker0 inet static
    bridge_ports eth0
    address 172.17.0.1
    netmask 255.255.0.0
```

Then I attempted to start docker with the following options configured in /etc/default/docker
```
DOCKER_OPTS="-b=docker0 --fixed-cidr=172.17.0.0/24"
```
docker fails to start and reports the following in the log:
```
INFO[0000] -job init_networkdriver() = ERR (1)          
FATA[0000] network does not contain specified subnet    
INFO[0000] +job serveapi(unix:///var/run/docker.sock)   
INFO[0000] Listening for HTTP on unix (/var/run/docker.sock) 
INFO[0000] +job init_networkdriver()                    
network does not contain specified subnet
```
I have two other machines configured in an identical manner, one with an ip of `172.17.1.1` and a fixed CIDR `172.17.1.0/24` and the other with an ip of `172.17.2.1` and fixed CIDR of `172.17.2.0/24` and when starting docker on those hosts there is no problem.

After double checking my configuration I experimented and updated the network and cidr:
```
auto docker0
iface docker0 inet static
    bridge_ports eth0
    address 172.17.3.1
    netmask 255.255.0.0
```
/etc/default/docker
```
DOCKER_OPTS="-b=docker0 --fixed-cidr=172.17.3.0/24"
```
Docker started without complaint. 
Is there something I'm missing here?

Docker details from `docker version`:
```
Client version: 1.4.1
Client API version: 1.16
Go version (client): go1.3.3
Git commit (client): 5bc2ff8
OS/Arch (client): linux/amd64
Server version: 1.4.1
Server API version: 1.16
Go version (server): go1.3.3
Git commit (server): 5bc2ff8
```

**Comment by estesp** [comment]
I was working on a solution to this bug, but then found #10215, which, if merged, will solve the problem anyway.  Waiting to hear the status of that PR before proceeding.

**Comment by jfrazelle** [comment]
closed becaue #10215 was merged
